common/hashicorp-vault/templates/vault-data.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.vault.server.sgx }}
```

Comment:
Why is this conditional?

Reply:
As this is part of the common chart, the idea behind it is to not include changes inside condition if no 'sgx' variable is defined. Let me know if this is the right approach from your perspective.

Reply:
This looks like the PVC that the vault chart already creates. I don't see what varies based on whether sgx is enabled or not? I could definitely be missing something here.

Reply:
Thanks for spotting this one! I removed the PVC. This definitely makes sense.